meta-ostro-xt/recipes-swupd/swupd-client/swupd-client_%.bbappend

```diff
@@ -1,10 +1,17 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
 SRC_URI_append = " \
+    file://efi_combo_updater.sh \
     file://ostroprojectorg.key \
 "
 
 do_install_append () {
+    rm -f ${D}/usr/bin/efi_combo_updater
+    install -m 0744 ${WORKDIR}/efi_combo_updater.sh ${D}/usr/bin/
+
+    rm -f ${D}/${systemd_system_unitdir}/efi-combo-trigger.service
+    install -m 0644 ${WORKDIR}/efi-combo-trigger.service.new ${D}/${systemd_system_unitdir}/efi-combo-trigger.service
```

Comment:
I consider this entire patchset to be temporary for the XT release and I expect the hack to be dropped once we can use again meta-ostro from ostro-os.

In the meanwhile I prefer to have more self-contained changes.